app/views/fact_check_confirm_error.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
```

Comment:
We need a test class for this view. As Mac kindly pointed out yesterday the coverage off command only disables the generated code.

Reply:
done, included in next commit